docs/implementing-cards.md

```diff
@@ -316,6 +316,35 @@ class SealOfTheHand extends DrawCard {
 }
 ```
 
+#### Ability messages
+
+The `message` property can be used to add a message to the game log outside of the `handler` function. By separating the message from the handler that executes the ability, messages can be added to the game log prior to prompting to cancel abilities (e.g. Treachery, Hand's Judgment, etc).
+
+The `message` property should be a function that takes an ability context object and returns an array with the message string as the first item in the array, followed by any arguments for the message:
+
+```javascript
+this.action({
+    // ...
+    message: context => ['{0} kneels {1} to stand {2}', context.player, this, this.parent],
```

Comment:
I'm not a huge fan of it needing to be an array, but I can't think of a better way..

`message: context => { format: 'blah blah', args: [1, 2, 3] }`

*might* be cleaner and more intuitive, but that's debatable.